explorer/nodewatch/nodewatch/RoutesFacade.py

```diff
@@ -78,6 +78,13 @@ def role_filter(descriptor):
 			lambda descriptor: descriptor.to_json(),
 			filter(role_filter, self.repository.node_descriptors)))
 
+	def json_node(self, filter_field, public_key):
+		"""Returns a node with matching public key."""
```

Comment:
It seems this function expects only a a public key 🤔

Reply:
might want to add this info

Reply:
Added 🙏🏼